UnityProjekte/Base Project/Assets/Scripts/HoverAnimation.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HoverAnimation : MonoBehaviour
+{
+    public float amplitude = 2f;
+    public float speed = 0.5f;
+
+    private void Update()
+    {
+        Vector3 p = transform.position;
+        p.y = amplitude * Mathf.Cos(Time.time * speed);
+        transform.position = p;
```

Comment:
the idea should work, simple mathematics, using the cos is a good idea! 👍 
(for better extensibility and readability you can put the code in a new function)